Entities/Timetable.cs

```diff
@@ -12,5 +12,17 @@ public Timetable()
         {
             Blocks = new List<Block>();
         }
+
+        public bool ContainsBlock(Block bl)
+        {
+            foreach (var blck in Blocks)
+            {
+                if(blck.IsSameAs(bl))
+                {
+                    return true;
+                }
```

Comment:
return blck,IsSameAs(b1);

Reply:
I cannot do that because I if I do that I will return first false